index.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+
+    <title>Document</title>
+</head>
+
+<body>
+    <header id="toppart">
+        <nav class="toolbar">
+            <a href="">
+                <img src="images/My Logo.svg" width="90" height="90">
+            </a>
+            <a href=""><img src="images/Hamburger button.svg" alt="three lines" width="24" height="24"></a>
+        </nav>
+
+    </header>
+
+    <section id="Introduction">
+        <h1>I'm Lethabo Glad to see you!</h1>
+        <div class="mid">
+            <p>I'm a software developer! I can help you build a product, feature or website Look through some fo my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        </div>
+        <hx class="center">LET'S CONNECT</hx>
+        <ul class="Socials">
+            <li>
+                <a href="#"><img src="images/Twitter.svg" alt="Twitter"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Linkedin.svg" alt="Linkedin"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Medium.svg" alt="Medium"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/GitHub.svg" alt="GitHub"></a>
+            </li>
+            <li>
+                <a href="#"><img src="images/Angel list.svg" alt="ANGEL LIST"></a>
+            </li>
+        </ul>
+    </section>
+    <section class="greybackground">
+    <section id="wrap">
+        <div class="whitebox">
+            <div class="imageclass">
+                <img src="images/orange ting.svg" height="250" />
+            </div>
+            <div class="description">
+                <h2 class="heading">Tonic</h2>
+
+                <div>
+                    <ul class="list_align">
+                        <li class="bold">CANOPY</li><img src="images/Counter.svg" />
+                        <li>Back End Dev</li><img src="images/Counter.svg" />
+                        <li>2015</li>
+                    </ul>
+                </div>
```

Comment:
- [ ] Kindly space horizontally in between these elements adequately like it is in the design.

- Your design
<img width="212" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227229940-dfee4c38-d299-42e6-aee4-c8f4dee95321.png">

- Figma design
<img width="191" alt="right" src="https://user-images.githubusercontent.com/41163571/227232500-f77fd55f-892c-4335-9307-6e3cf38db5c2.png">